webpush/admin.py

```diff
@@ -1,3 +1,17 @@
 from django.contrib import admin
 
-# Register your models here.
+from .models import PushInformation
+from .utils import _send_notification
+
+
+class PushInfoAdmin(admin.ModelAdmin):
+    list_display = ("__str__", "user", "subscription", "group")
+    actions = ("send_message",)
+
+    def send_message(self, request, queryset):
+        result = []
+        payload = "{\"head\": \"Hey\", \"body\": \"Hello World\"}"
```

Comment:
Can you please make a pyhton ``dictionary`` and then make it json with ``json.dump()``? So it will look more conventional.